app/views/devise/sessions/new.html.erb

```diff
@@ -2,39 +2,36 @@
   <div>
     <%= image_tag 'logo.svg', alt: 'LNU logo', width: 200, height: 200, class: 'mx-auto h-12 w-auto' %>
     <h2 class="mt-6 text-center text-3xl font-extrabold text-gray-900">
-       <%=  t('devise.sessions.user.heading_new') %>
+       <%=  t('simple_form.labels.user.session.heading_new') %>
     </h2>
     <p class="mt-2 text-center text-sm text-gray-600">
-       <%=  t('devise.registrations.user.or') %>
-      <%= link_to t('devise.sessions.user.register_instead'), new_registration_path(resource_name), class: 'font-medium text-lnu-pink hover:text-lnu-blue' %>
+       <%=  t('simple_form.labels.user.or') %>
+      <%= link_to t('simple_form.labels.user.register_instead'), new_registration_path(resource_name), class: 'font-medium text-lnu-pink hover:text-lnu-blue' %>
     </p>
   </div>
 
-  <%= form_for(resource, as: resource_name, url: session_path(resource_name), class: 'mt-8 space-y-6') do |f| %>
+  <%= simple_form_for(resource, as: resource_name, url: session_path(resource_name), class: 'vertical_form') do |f| %>
     <input type="hidden" name="remember" value="true">
     <div class="rounded-md shadow-sm -space-y-px">
       <div>
-        <%= f.label :email, t('devise.registrations.user.email'), class: 'sr-only' %>
-        <%= f.email_field :email, autocomplete: 'email', class: 'fieldFirst textField', placeholder: 'epost@epost.no', autofocus: true, required: true %>
+        <%= f.input :email, input_html: { class: "fieldFirst" }, autocomplete: 'email', placeholder: t("simple_form.labels.user.email_example"), autofocus: true, label_html: { class: "sr-only"} %>
       </div>
       <div>
-        <%= f.label :password, t('devise.registrations.user.password'), class: 'sr-only' %>
-        <%= f.password_field :password, autocomplete: "current-password", class: 'passwordField fieldLast', placeholder: '******', required: true %>
+        <%= f.input :password, input_html: { class: "fieldLast" }, autocomplete: "current-password",  placeholder: '******', required: true, label_html: { class: "sr-only" } %>
       </div>
     </div>
     <div class="flex items-center justify-between mt-4">
       <div class="flex items-center">
         <% if devise_mapping.rememberable? %>
-          <%= f.check_box :remember_me, class: "h-4 w-4 text-lnu-pink focus:ring-lnu-pink border-gray-300 rounded" %>
-          <%= f.label :remember_me, t('devise.sessions.user.remember_me'), class: 'ml-2 block text-sm text-gray-900' %>
+          <%= f.input :remember_me, as: :boolean %>
```

Comment:
Denne ser ut til å mangle class fra tidligere `class: 'ml-2 block text-sm text-gray-900`, så blir skjevt i skjema: 

![image](https://user-images.githubusercontent.com/14905290/141182017-3ef6f943-3557-465e-9cd3-3fca5f16d745.png)

Men: Når jeg prøver legge det til igjen så skjer det ingen forskjell - så veldig usikker på hvor denne tar styling fra, og hvorfor det er skjevt nå.

Reply:
Det legges inn i config/initializers/simple_form_tailwind.rb:62

De klassene er lagt inn allerede. Men om du skal legge inn flere klasser på simple_form, som ikke skal som standard på en input type så kan du bruke `input_html: { class: "classene" }` også for labels `input_label: { class: "classes" }`

Men har ikke gjort noe med de annet enn å legge de inn da jeg regnet med at du fikser nytt design etterhvert :+1: 
Det kan være noe greier som påvirkes av de andre simple_form stylene som gjør dette. 
Vi må gå over alle stylene og sette defaults.